omero/sysadmins/limitations.txt

```diff
@@ -193,7 +193,8 @@ LDAP
 
 Enabling synchronization of LDAP on user login will result in LDAP
```

Comment:
I think woudl be good idea to link it to http://www.openmicroscopy.org/site/support/omero5.1/sysadmins/server-ldap.html#synchronizing-ldap-on-user-login and from